javascript/ql/src/Security/CWE-094/CodeInjection.qhelp

```diff
@@ -32,6 +32,23 @@ for example, steal cookies containing session information.
 <sample src="examples/CodeInjection.js" />
 </example>
 
+<example>
```

Comment:
There are 3 examples split across 2 separate "Example" sections. It might be best to put them all under one single "Examples" heading. What do you think?

Additionally, it would be nice to add a solution or workaround to the new example you added, or is it obvious to folks?

Reply:
👍  Merged the two sections.

The solution to the new example is in the second paragraph below.